src/dbos.ts

```diff
@@ -404,6 +406,7 @@ export class DBOS {
       options.honoApp.use(honoTracingMiddleware);
     }
 
+    DBOSExecutor.globalInstance.createInternalQueue();
```

Comment:
I think we should create the internal queue earlier before DBOS initializes. In the test I see a lot of warning messages: `[warn]: Workflow queue '_dbos_internal_queue' is being created after DBOS initialization and will not be considered for dequeue.`